tests/libwnbd_tests/scsi_helper.cpp

```diff
@@ -0,0 +1,52 @@
+#include "scsi_helper.h"
+
+void ScsiPassthroughHelper::PersistentReserveIn() {
+	UCHAR cdb[16];
```

Comment:
It's true that wnbd doesn't parse scsi per rsv commands but we should probably ensure that it gets transmitted properly.